code/game/objects/items/defibrillator.dm

```diff
@@ -140,6 +139,12 @@
 			return ghost
 	return null
 
+///GET IN DA BODY
+/mob/living/proc/grab_ghost()
+	var/mob/dead/observer/G = get_ghost()
+	. = G
+	if(G)
+		G.reenter_corpse()
```

Comment:
Do you really need a proc for this? Its just an extra proc call.

Reply:
yeah

Reply:
just call reenter_corpse

Reply:
no